src/Microsoft.Azure.WebJobs.Host/Properties/AssemblyInfo.cs

```diff
@@ -8,6 +8,7 @@
 [assembly: InternalsVisibleTo("Dashboard.UnitTests")]
 [assembly: InternalsVisibleTo("DynamicProxyGenAssembly2")]
 [assembly: InternalsVisibleTo("Microsoft.Azure.WebJobs.Host.TestCommon")]
+[assembly: InternalsVisibleTo("Microsoft.Azure.WebJobs.Host.EndToEndTests")]
```

Comment:
We've tried to avoid this since the E2E tests are supposed to be how customers would be able to interact with our stuff. Is there something that we need to expose so we can remove this?

Reply:
Yeah, didn't like that either. This was to have access to the newly introduced `Utility` class. I don't really want to make that public.

We could remove the `InternalsVisibleTo` and link to that file. That would prevent us from inadvertently use internal types in tests.